packages/grpc-native-core/ext/call_credentials.cc

```diff
@@ -245,11 +245,10 @@ int plugin_get_metadata(
     grpc_metadata creds_md[GRPC_METADATA_CREDENTIALS_PLUGIN_SYNC_MAX],
     size_t *num_creds_md, grpc_status_code *status,
     const char **error_details) {
+  HandleScope scope;
```

Comment:
I don't think we need a `HandleScope` here because this function doesn't call any Node APIs.

Reply:
other reason: `tag` represents an async resource. So it makes sense to be a subclass of `AsyncResource`.

Reply:
Fix in Nan: https://github.com/nodejs/nan/pull/757 which will take some time to review / land / release. For now let's use a HandleScope ourselves.